### DISCUSSION TITLE
The terminal will have problems

### DISCUSSION THREAD

**Discussion by Half-Sugar-Code** [SBoudrias/Inquirer.js#1156] [General]
hi:
When I will inquirer type set to rawlist or list, After running the command, when the MAC terminal presses the up and down arrows, the terminal output is ^ [[a ^ [[b

**Comment by SBoudrias** [comment]
Can you post your full code? This is likely due to conflict with other things you're doing.

Did you try running the examples from the inquirer repository?

**Reply by Half-Sugar-Code** [reply]
Thank you. Later, I found that such problems occur when I use nodemon to run files. It is normal when I use node to run files.

**Reply by SBoudrias** [reply]
Ah, I see. You need to make sure you inherit the stdio to the nodemon underlying process. Otherwise inquirer doesn't receive an interactive prompt, so it cannot listen for keyboard events.

**Reply by Half-Sugar-Code** [reply]
OK, I see. Thank you for your reply.

**Reply by Half-Sugar-Code** [reply]
@SBoudrias. Hi I have another question. How can I change the default prompt text after the prompt message when the type is checkbox?
Message Text: Press <space> to select, <a> to toggle all, <i> to invert selection)

**Reply by Half-Sugar-Code** [reply]
Hi, @SBoudrias, Do you have any reference documents for my above questions?

**Reply by SBoudrias** [reply]
You could [hide the default help message](https://github.com/SBoudrias/Inquirer.js/blob/master/packages/inquirer/lib/prompts/checkbox.js#L89) and add a custom one in your `message` string.

**Reply by Half-Sugar-Code** [reply]
I just checked the source code of the part you marked for me. I understand whether the value of dontShowHints should be obtained from this.opt?

**Reply by SBoudrias** [reply]
Yeah, you'll just want to pass `dontShowHints` as a flag to `inquirer.prompt([{ type: 'checkbox', dontShowHints: true, ...etc }])`

**Reply by Half-Sugar-Code** [reply]
hi, @SBoudrias, This is my example.
```js
import inquirer from "inquirer";
inquirer
  .prompt([
    {
      type: 'checkbox',
      message: 'Select toppings',
      name: 'toppings',
      dontShowHints:true,
      choices: ['A',"B",'C']
    },
  ])
  .then((answers) => {
    console.log(JSON.stringify(answers, null, '  '));
  });

```
But after I run it with node, it doesn't work.
I think this code is right, are you think?
```js
 if (!this.dontShowHints) { // I think this.dontShowHints should be changed to this.opt.dontShowHints
      message +=
        '(Press ' +
        chalk.cyan.bold('<space>') +
        ' to select, ' +
        chalk.cyan.bold('<a>') +
        ' to toggle all, ' +
        chalk.cyan.bold('<i>') +
        ' to invert selection, and ' +
        chalk.cyan.bold('<enter>') +
        ' to proceed)';
    }
```

**Reply by SBoudrias** [reply]
Can you clarify what "it doesn't work" exactly mean? Nothing happens? An error is thrown?

Also, please provide details of how you're running the script. Is it just `node myfile.js`? Or are you using a tool like `nodemon`, etc.

Without any of those details, it is very hard to even start to help.

**Reply by Half-Sugar-Code** [reply]
I used `node myfile.js`. The effect is that I added the following configuration

```js

dontShowHints = true

```

However, the default value of type = checkbox still appears.

default value is :Message Text: Press to select, to toggle all, to invert selection)

**Reply by SBoudrias** [reply]
Did you double check your inquirer version? This is a fairly new feature

**Reply by Half-Sugar-Code** [reply]
I inquirer version is `inquirer@9.1.0`

**Reply by Half-Sugar-Code** [reply]
@SBoudrias Sorry to bother you again. Can you give me an example code that cancels the default value of the `checkbox` option? As you said, adding `dontShowHints = true` configuration does not cancel the default value of the `checkbox` option
